docs/iot/iot-partners/index.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with Nubison Cloud
+
+The Nubison cloud can readily be interconnected with Tizen IoT devices, which has an intrinsic capability of seamless connectivity.
+Real-time data monitoring and controlling between Tizen service app and Nubison cloud is empowered through Nubison plugin.
+
+**Figure: Architecture**
+
+![Architecture](media/architecture.png)
+
+Compatibility
+- Platform : Tizen 5.0 IoT Headed & Headless and later
+- SDK : Tizen Studio 3.0 and later 
```

Comment:
The Nubison plugin is installed on Tizen IoT devices compatible with as follow: 
   - Platform : Tizen 5.0 IoT Headed & Headless and later
   - SDK : Tizen Studio 3.0 and later

Reply:
The Nubison plugin is NOT installed on Tizen IoT devices.
I will change it to 'The Nubison plugin can be installed...'

Reply:
@bson1012 

The Nubison plugin can be installed on Tizen IoT devices that meet the following compatibility criteria:

    * Platform : Tizen 5.0 IoT Headed & Headless and later

    * SDK : Tizen Studio 3.0 and later